File: Java/bee 1009.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String nome;
        double salario;
        double vendas;
        //comissao de 15%

        nome = sc.next();
        salario = sc.nextDouble();
        vendas = sc.nextDouble();

        System.out.printf("TOTAL = R$ %.2f\n", salario + vendas * 0.15);
    }
}

File: Kotlin/bee 1009.kt
import java.util.*
import java.io.BufferedReader
import java.io.InputStreamReader

fun main(args: Array<String>) {

    val br = BufferedReader(InputStreamReader(System.`in`))
    val nome = br.readLine()
    val salario = br.readLine().toDouble()
    val comissao = br.readLine().toDouble() * 0.15

    println(String.format("TOTAL = R$ %.2f", (salario + comissao)))
}
